package.json

```diff
@@ -1,7 +1,7 @@
 {
   "scripts": {
     "clean": "rm -rf node_modules yarn-error.log css static/css *.log *.sqlite _site/ build/ .jekyll-metadata .bundle",
-    "serve": "./entrypoint 0.0.0.0:$PORT",
+    "start": "./entrypoint 0.0.0.0:$PORT",
```

Comment:
I think keeping both is neater even if in this case they do the same, `start` can be:
"start": "yarn run serve"

Reply:
:+1:  package.json updated